uiowa/uiowa
# update cshs to supported version

I am getting an install error for a website related to this module. Try updating to the supported 2.x branch.

https://github.com/uiowa/uiowa/issues/3189#issuecomment-791772487

## To Test

Make sure on a node edit form the menu settings stuff still works as expected. (e.g. changing parent, dragging order)
